pyproject.toml

```diff
@@ -7,11 +7,11 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-teslajsonpy = "^1.4.1"
+teslajsonpy = "^1.4.2"
 
 [tool.poetry.dev-dependencies]
-homeassistant = "^2021.3.4"
-pytest-homeassistant-custom-component = "~=0.3.1"
+homeassistant = "=2021.6.0b0"
+pytest-homeassistant-custom-component = "=0.4.0"
```

Comment:
I wonder if this change is what is breaking pytest.

Reply:
LIkely. I had to update to at least that version of homeassistant though for this function https://github.com/home-assistant/core/pull/49912

Reply:
BTW, this was confirmed to be the cause. https://github.com/alandtse/tesla/issues/170

Reply:
As a note, I figured out the fix if you ever want to pick this up again.

Reply:
@alandtse I actually closed this because it looks like this was kinda implemented in the latest version with the Poll switch.